automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -239,9 +239,26 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
           }
         }
       }
-
     }
 
+    "should install multiple user specified notebook extensions" in withWebDriver { implicit driver =>
+      val clusterName = ClusterName("user-jupyter-ext" + makeRandomId())
+      withCleanBillingProject(hermioneCreds) { projectName =>
```

Comment:
You can reduce some of this boiler plate (lines 246-249) by rebasing your branch and using Rob's `withProject` method, added with in [this PR](https://github.com/DataBiosphere/leonardo/pull/331/files#diff-7d71faa33c09b3f5553be71dd25769d5L43).